src/components/Options/StudentDetailsOptions.js

```diff
@@ -0,0 +1,178 @@
+const programOptions = [
```

Comment:
Why is this hardcoded. @suryabulusu ?

Reply:
this is like the `constants.js` file we have in `college-predictor`. https://github.com/avantifellows/college-predictor/blob/main/constants.js

idea is to keep all possible options in single js file so that its easy to edit.